interp/interp.go

```diff
@@ -670,10 +670,17 @@ func (r *Runner) redir(rd *syntax.Redirect) (io.Closer, error) {
 	case syntax.RdrOut, syntax.RdrAll:
 		mode = os.O_RDWR | os.O_CREATE | os.O_TRUNC
 	}
-	f, err := os.OpenFile(r.relPath(arg), mode, 0644)
-	if err != nil {
-		// TODO: print to stderr?
-		return nil, err
+	var f io.ReadWriteCloser
+	switch arg {
+	case "/dev/null":
```

Comment:
Perhaps we should use something like `os.Stat` to make symlinks and stuff like `/dev/../dev/null` work, even if they're a bit silly. But this is fine for the first version.